scripts/test_duplicate_prevention.py

```
@@ -0,0 +1,219 @@
+        print("📊 중복 메시지 통계:")
+        total_duplicates = 0
+        for row in rows:
+            count = row['count']
+            if count > 1:
+                total_duplicates += count - 1
+                print(f"  ⚠️  중복: '{row['content'][:50]}...' ({count}개)")
+                print(f"      메시지 IDs: {row['message_ids']}")
+            else:
+                print(f"  ✅  정상: '{row['content'][:50]}...' (1개)")
+        
+        print(f"\n📈 전체 통계:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)